.travis.yml

```diff
@@ -0,0 +1,20 @@
+# Config file for automatic testing at travis-ci.org
+
+sudo: false
+
+language: python
+
+matrix:
```

Comment:
Which confused me at first because those `TOXENV` values didn't match anything in the tox.ini `envlist`, but I guess they don't strictly need to match.  Updating here to follow suit.